environment-dev.yml

```diff
@@ -6,48 +6,42 @@ dependencies:
 
   # required dependencies
   - pandas>=2,<2.1
-  - numpy>=1.18.5
-  - fsspec
-  - packaging
-  - psutil
+  - numpy>=1.20.3
+  - fsspec>=2021.07.0
+  - packaging>=21.0
+  - psutil>=5.8.0
 
   # optional dependencies
   # ray==2.5.0 broken: https://github.com/conda-forge/ray-packages-feedstock/issues/100
   - ray-default>=1.13.0,!=2.5.0
   # https://github.com/modin-project/modin/issues/6336
   - pydantic<2
-  - pyarrow
+  - pyarrow>=7.0.0
```

Comment:
this is pretty aggressive bump, isn't it? why cannot we use earlier verions?

Reply:
Looks like `pandas 2.*` isn't compatible with `pyarrow 6.*`
https://github.com/modin-project/modin/actions/runs/5649681599/job/15304749494